compiler/src/main/java/com/squareup/anvil/compiler/codegen/dagger/AnvilAnnotationDetectorCheck.kt

```diff
@@ -14,8 +17,11 @@ import org.jetbrains.kotlin.descriptors.ModuleDescriptor
 import org.jetbrains.kotlin.psi.KtFile
 import java.io.File
 
+@AutoService(CodeGenerator::class)
```

Comment:
Should the internal code generators use this mechanism? Would that have a performance impact if no external code generator is used (98% of the cases)? Or wouldn't it matter, because we need to check for external generators anyways?

Reply:
Wouldn't matter because you'd need to check for external generators anyway!